pynance/tst/unit/learn/x.data

```diff
@@ -0,0 +1,12 @@
+ -15.93675813378541
```

Comment:
I considered that. If we end up with a lot of tests that rely on data sets that need to be read from files, then I think we should do that. But I don't expect that to be the case. Usually, we can just use toy data, as in your integration test. Within the directory, it's easy to distinguish between actual test files (which have to begin with 'test' for nose to run them), other python files (ending in `.py` but not starting with `test`) and data files (labeled `data`).

Currently, this is the only directory that even *has* this kind of structure, and it's very easy to see which files are which in light of naming. Moving to a different directory would also make the coding slightly more convoluted in that the paths wouldn't be local. I agree with you if this kind of things grows more but feel like it makes things simpler for now to put the data with the test that draws from it. Moreover, factoring out a separate directory suggests that it might be data used by more than 1 test. That isn't the case, and it's doubtful it ever will be.